### PR TITLE
178 check if rust can be added to sdks

### LINKED ISSUE #178: Check if Rust can be added to SDKs

### PR DESCRIPTION
**Pull request by jonas089**
I added Rust examples utilising the casper-client crate alongside the Javascript and Python SDK examples. I tried to keep it simple and follow a similar structure. I tested all functions and everything looks good on my end.

### CODE REVIEW

**Comment by jonas089** [comment]
@melpadden @ACStoneCL @KMCreatesWorlds  it would be great if either of you could review my contribution. Thanks in advance :)

File: source/docs/casper/developers/dapps/sdk/client-library-usage.md [hunk 1]
```diff
@@ -30,6 +30,38 @@ pip install pycspr
...
+casper-client="1.5.1"
+```
+
+and add it to your `main.rs`:
+
+```rust
+extern crate casper_client;
+```
+
+use types and methods from casper_client:
```
Comment (KMCreatesWorlds) [inline source/docs/casper/developers/dapps/sdk/client-library-usage.md]:
```suggestion
Use types and methods from casper_client:
```

File: source/docs/casper/developers/dapps/sdk/client-library-usage.md [hunk 2]
```diff
@@ -270,6 +362,43 @@ print(deploy.hash.hex())
...
+        ttl:"50s",
+        gas_price:"1000000000",
+        chain_name:"casper", // or "casper-test"
+        dependencies: Vec::new(),
+        session_account: "01daad67ebbcb725e02a1955a6617512b311435a21ca6d523085aa015d2d1b473a"
+
+    };
+    // without session args:
+    //let session_args: Vec<&str> = Vec::new();
+    // with session args:
```
Comment (KMCreatesWorlds) [inline source/docs/casper/developers/dapps/sdk/client-library-usage.md]:
```suggestion
    // Without session args:
    // let session_args: Vec<&str> = Vec::new();
    // With session args:
```

File: source/docs/casper/developers/dapps/sdk/client-library-usage.md [hunk 3]
```diff
@@ -364,6 +493,44 @@ print(deploy.hash.hex())
...
+        dependencies: Vec::new(),
+        session_account: "01daad67ebbcb725e02a1955a6617512b311435a21ca6d523085aa015d2d1b473a"
+
+    };
+    let mut session_args: Vec<&str> = Vec::new();
+    session_args.push("amount:U512='500000000000'");
+    
+    session_args.push("delegator:public_key='01daad67ebbcb725e02a1955a6617512b311435a21ca6d523085aa015d2d1b473a'");
+    session_args.push("validator:public_key='validator_public_key'");
+    // test with session args -> tbd
```
Comment (KMCreatesWorlds) [inline source/docs/casper/developers/dapps/sdk/client-library-usage.md]:
```suggestion
  
```

**Review by KMCreatesWorlds — CHANGES_REQUESTED** [review]
The general usage of the SDK is correct, only small suggestions to the comments etc. otherwise it looks good. After the commit of the suggestions it can be approved and merged.

**Review by KMCreatesWorlds — APPROVED** [review]

**Review by ghost — APPROVED** [review]
I am giving approval, just to allow merge after Karol's review :rocket:  - he is great member of Tiger team, but since he does not belong to [Casper Association organization on GitHub](https://github.com/orgs/casper-network/teams), we cannot add him directly to [tiger-team](https://github.com/orgs/casper-network/teams/tiger-team/members) (code owner here).

**Pull request merged**